authors/apps/articles/utils/model_helpers.py

```diff
@@ -0,0 +1,10 @@
+from ..models import Article
+
+def get_single_article_using_slug(slug):
+        # This method will be used to get a single article using a slug
+        # It will return a none is it tries getting an article with non exisiting slug
+        try:
+            obj = Article.objects.get(slug=slug)
+            return obj
+        except Article.DoesNotExist:
+            return None
```

Comment:
Hi @FahdJamy, great work man, you might want to take care of the End of Line after line 10

Reply:
Thanks. let me work on that now